app/controllers/miq_request_controller.rb

```diff
@@ -415,16 +415,26 @@ def get_request_tab_type
     end
   end
 
+  def remote_and_global_requestors(requestor)
```

Comment:
@yrudman @lpichler I definitely like the suggestions but, let's make the suggested changes in a followup PR since we need to get this one fix quickly.